package.json

```diff
@@ -26,7 +26,7 @@
     "web-vitals": "^1.1.0"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "react-scripts --openssl-legacy-provider start",
```

Comment:
What dis for?

Reply:
my `npm start` commands fails if I don't change this start script for whatever reason, might just be a me thing 🤷

Reply:
https://stackoverflow.com/a/69713899

Reply:
⬆️ this is the SO Q&A I followed